### DISCUSSION TITLE
配置网址是填pc端官网的还是移动端分享的链接

### DISCUSSION THREAD

**Discussion by Hanabiiiiii** [mikumifa/biliTickerBuy#230] [提问专区]
![image](https://github.com/mikumifa/biliTickerBuy/assets/122472858/5c3a19e3-6b0c-4c4a-aa63-933fa09413b7)
![image](https://github.com/mikumifa/biliTickerBuy/assets/122472858/8a0278f3-2b73-4d6f-8631-e8691cfb8c23)

**Comment by WittF** [comment]
BW 
https://show.bilibili.com/platform/detail.html?id=85939

BML 
https://show.bilibili.com/platform/detail.html?id=85938
